content/posts/newsletter-015/index.md

```diff
@@ -668,6 +668,47 @@ on the module's [repository][Proton-Media-Converter-Github]
 [Proton-Github]: https://github.com/ValveSoftware/Proton
 [Proton-Media-Converter-Github]: https://github.com/ValveSoftware/Proton/tree/proton_5.13/media-converter
 
+### [Rust GPU][RustGPU-Github] v0.1
+
+![Rust GPU Sky](rustgpu.jpg)
+_Sky example in Rust GPU_
+
+Rust GPU is a very early stage project to make Rust a first-class language and
+ecosystem for building GPU code.
+
+[Rust GPU released v0.1 in October][RustGPU-Twitter], and has already garnered
```

Comment:
Although still in very early stages of development, [Rust GPU released v0.1 in October][RustGPU-Twitter], and has already garnered....

[RustGPU-Twitter]: https://twitter.com/repi/status/1319274584915365888

Reply:
:100: